docs/introduction.md

```diff
@@ -0,0 +1,29 @@
+---
+id: introduction
+slug: /
+---
+
+# Introduction
+
+Jellyfish is a WebRTC Media Server implemented in Elixir.
+It allows for effortless creationg of online meeting rooms.
```

Comment:
Very laconic. You can probably say a bit more about what it is and what it will provide.
Or you can do something like in [LiveKit Docs](https://docs.livekit.io/), where landing page (where they say that their product is great etc.) is different from introduction/getting started (more like real "documentation", with facts, not opinions if you know what I mean).

Reply:
I am not sure if we need a separate landing page, for now the introduction can be our landing page.
I will make the description more meaningful.